src/data/auth.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Constants used by authentication.
+ */
+
+const retroCertsBasePath = "/retroactive-certification";
+
+const auth = {
+  AUTHTOKEN: "authToken", // Session storage key.
+  apiPath: {
+    login: retroCertsBasePath + "/api/login",
+    data: retroCertsBasePath + "/api/data",
+  },
+  statusCode: {
+    OK: "ok",
+    notLoggedIn: "not-logged-in",
+    userNotFound: "user-not-found",
+    wrongEddcan: "wrong-eddcan",
+    wrongSsn: "wrong-ssn",
+  }
+}
+
+module.exports = auth;
```

Comment:
what do you think of using CONSTANT_CASE across the codebase? and here, making auth AUTH_STRINGS (object keys don't need to change)

i'd find the index.js easier to read if it was like AUTH_STRINGS.apiPath.data rather than auth.apiPath.data, then i don't have to wonder if data might contain something 

from https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
I like that much better. I renamed the export to AUTH_STRINGS and made all the keys camelCase. I also renamed the file to authStrings.js.